components/checkbox/checkbox.js

```diff
@@ -8,16 +8,24 @@ function Checkbox(props) {
   const {
     checked,
     children,
+    dataAttrs = {},
     disabled,
     name,
     onChange,
+    ...otherProps
```

Comment:
why then you need `dataAttributes` (which were created exactly for `data-gtm-id` and other data attributes)?